src/main/java/smokefree/domain/Initiative.java

```diff
@@ -253,6 +255,21 @@ private void validatePlaygroundsRange(GeoLocation newPlaygroundLocation, long di
                             "playground does already exists within "+ SmokefreeConstants.MAXIMUM_PLAYGROUNDS_DISTANCE+ " Meters");
                 });
     }
+    private void validateMaximumAllowedVolunteers() {
+        if(citizens.stream().count() >= SmokefreeConstants.PlaygroundWorkspace.MAXIMUM_VOLUNTEERS_ALLOWED) {
+            throw new DomainException("MAXIMUM_VOLUNTEERS",
+                    "Maximum " + SmokefreeConstants.PlaygroundWorkspace.MAXIMUM_VOLUNTEERS_ALLOWED + " members can join the initiative" ,
+                    "Sorry, only " + SmokefreeConstants.PlaygroundWorkspace.MAXIMUM_VOLUNTEERS_ALLOWED + " members can join the initiative");
+        }
+    }
+
+    private void validateMaximumAllowedManagers() {
+        if(managers.stream().count() >= 2) {
+            throw new DomainException("MAXIMUM_MANAGERS",
+                    "Maximum " + SmokefreeConstants.PlaygroundWorkspace.MAXIMUM_MANAGERS_ALLOWED + " volunteers can claim for manager role",
```

Comment:
I would phrase this as:
'No more than [...] volunteers can claim the manager role'

Reply:
made changes as suggested